hpat/tests/test_series.py

```diff
@@ -3373,6 +3374,71 @@ def test_impl(s, axis):
             msg = 'Method cumsum(). Unsupported parameters. Given axis: int'
             self.assertIn(msg, str(raises.exception))
 
+    def test_series_pct_change(self):
+        def test_series_pct_change_impl(S, periods, method):
+            return S.pct_change(periods=periods, fill_method=method, limit=None, freq=None)
+
+        hpat_func = hpat.jit(test_series_pct_change_impl)
+        test_input_data = [
+            [],
+            [np.nan, np.nan, np.nan],
+            [np.nan, np.nan, np.inf],
+            [0] * 8,
+            [0, 0, 0, np.nan, np.nan, 0, 0, np.nan, np.inf, 0, 0, np.inf, np.inf],
+            [1.1, 0.3, np.nan, 1, np.inf, 0, 1.1, np.nan, 2.2, np.inf, 2, 2],
+            [1, 2, 3, 4, np.nan, np.inf, 0, 0, np.nan, np.nan]
+        ]
+        for input_data in test_input_data:
+            S = pd.Series(input_data)
+            for periods in [0, 1, 2, 5, 10, -1, -2, -5]:
+                for method in [None, 'pad', 'ffill', 'backfill', 'bfill']:
+                    result_ref = test_series_pct_change_impl(S, periods, method)
+                    result = hpat_func(S, periods, method)
+                    pd.testing.assert_series_equal(result, result_ref)
+
+    def test_series_pct_change_str(self):
+        def test_series_pct_change_impl(S):
+            return S.pct_change(periods=1, fill_method='pad', limit=None, freq=None)
+
+        hpat_func = hpat.jit(test_series_pct_change_impl)
+        S = pd.Series(test_global_input_data_unicode_kind4)
+
+        with self.assertRaises(TypingError) as raises:
+            hpat_func(S)
+        msg = 'Method pct_change(). The function only applies to elements that are all numeric. Given data type: unicode_type'
+        self.assertIn(msg, str(raises.exception))
+
+    def test_series_pct_change_not_supported(self):
```

Comment:
I think you should skip the test in hpat pipeline via decorator:
`@unittest.skipIf(hpat.config.config_pipeline_hpat_default, 'Series.pct_change() does not raise an exception')`